package/Dockerfile

```
@@ -6,7 +6,7 @@ RUN make -C /helm
 FROM registry.suse.com/bci/bci-base:15.4.27.14.21 AS build
 ARG ARCH=amd64
 RUN zypper -n install curl gzip tar
-ENV KUBECTL_VERSION v1.21.8
+ENV KUBECTL_VERSION v1.23.14
```

@TomyLobo can you please bump to [v1.24.9](https://github.com/kubernetes/kubectl/releases/tag/v0.24.9) and then we can merge it, please?

```suggestion
ENV KUBECTL_VERSION v1.24.9
```